### DISCUSSION TITLE
API access to all seller Reports available in Seller Central

### DISCUSSION THREAD

**Discussion by kevnhowe** [amzn/selling-partner-api-models#4164] [Ideas] [answered]
Please build all available seller central reports in to the SPAPI. I imagine that seller central accesses the report data via an API of sorts, so why not enable them for us to interface programmatically?
Brand Analytics are sorely missing from the API. 
Top Search Terms
Demographics
Search Catalog Performance 
Search Query Performance
Repeat Purchase Behavior 
These are just a few of the reports we need to close the gaps in our analytics workflow.

**Comment by mafge** [comment, marked as answer]
Hi @kevnhowe,
thanks for reaching out!
Have you checked out the available [brand analytics reports](https://developer-docs.amazon.com/sp-api/docs/report-type-values-analytics#brand-analytics-reports)? Looks like some of them could fit your needs.
Please also note that we won't add new reports to Reports API. In the future all data will be available via [Data Kiosk](https://developer-docs.amazon.com/sp-api/docs/data-kiosk-api-v2023-11-15-reference). Currently only a limited set of data is available, but this will change step by step in the future.
Best,
Marc
Selling Partner Developer Services

**Reply by kevnhowe** [reply]
I have looked into the reports and only market basket meets our requirements.
That is great news on Data Kiosk. Will existing reports be moved over to Data Kiosk? It is very fast when compared to other report requests. 
Do you have a roadmap available for the API? I would love to be able to plan implementations around planned changes and releases on the API.
Also, is the API team going to be at Accelerate? I'd love to be able to connect and hear what you all have to say about where we are at and where SP API is going.

**Reply by mafge** [reply]
@kevnhowe We will make new datasets available in the coming months, unfortunately there is no timeline that I can share by now. I recommend to check our [changelog](https://developer-docs.amazon.com/sp-api/changelog) from time to time or subscribe to the [RSS feed](https://developer-docs.amazon.com/sp-api/changelog.rss).

Accelerate is mainly focused on sellers, not so much SP API. We usually host our own events for invited developers where we focus purely on the API.
